x-pack/plugins/security_solution_ess/public/upselling/pages/entity_analytics_upselling.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import {
+  EuiCard,
+  EuiIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiButton,
+  EuiTextColor,
+  EuiImage,
+  EuiPageHeader,
+  EuiSpacer,
+} from '@elastic/eui';
+// eslint-disable-next-line @kbn/eslint/module_migration
```

Comment:
nit: why not just use emotion? If you want to keep the same pattern you can just import from `@emotion/styled`.

Reply:
Wow, I didn't know that `@emotion/styled` was a thing. Thank you!
TIL